packages/fxa-auth-server/package.json

```diff
@@ -29,7 +29,11 @@
     "paypal-processor": "CONFIG_FILES='config/secrets.json' ts-node ./scripts/paypal-processor.ts",
     "subscription-reminders": "CONFIG_FILES='config/secrets.json' ts-node ./scripts/subscription-reminders.ts",
     "write-templates": "ts-node lib/senders/emails/render-to-disk.ts",
-    "maildev-preview": "ts-node lib/senders/emails/preview-in-maidev.ts"
+    "maildev-preview": "ts-node lib/senders/emails/preview-in-maidev.ts",
+    "email-scss": "node-sass lib/senders/emails/scss -o lib/senders/emails/css",
+    "start-mjml": "ts-node bin/mjml-sever.ts",
+    "storybook": "ts-node lib/senders/emails/sass-compile-files.ts && start-storybook -p 6007 --no-version-updates",
+    "storybook-preview": "npm-run-all -p start-mjml storybook"
```

Comment:
Renaming the `mjml-server.ts` file and changing the storybook port is being taken care of in the other PR